index.html

```diff
@@ -54,11 +58,173 @@ <h1>I’m Sri Sudha Suryadevara.
                 </div>
             </div>
         </section>
+        <!-- end of Banner -->
 
-        <section class="about-container">
+        <!-- start of works -->
+        <section class="work">
+            <div class="container">
+                <div class="card">
+                    <div class="card-left">
+                        <img src="Images/work/Snapshoot Portfolio.png" alt="snap-card">
+                    </div>
+                    <div class="card-right">
+                        <h2>Tonic</h2>
+                        <ul class="cano">
+                            <li>CANOPY<i class="fa fa-circle"></i></li>
+                            <li>Back End Dev<i class="fa fa-circle"></i></li>
+                            <li>2015</li>
+                        </ul>
+                        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                        <ul class="lang">
+                            <li><a href="#">HTML</a></li>
+                            <li><a href="#">CSS</a></li>
+                            <li><a href="#">Javascript</a></li>
+                        </ul>
+                        <button class="btn-default">See project</button>
+                    </div>
+                </div>
```

Comment:
- [ ] Excellent job done so far 👏 I think it would be better to add a bit more space between the elements for getting a design close to the figma template as much as possible as suggested [here](https://github.com/Srilearns/Portfolio-Website-Mobile-First/pull/3#discussion_r1256911920)
- [ ] Also, as suggested, kindly make sure to adjust the `font-size`. The project's title is looking a bit bigger from the given design ✨ In Figma template, the `font-size` for project's title is `32px` while yo are using `40px`
- [ ] I noticed that you are using `box-shadow` for cards. I appreciate your efforts, however in Figma template, box shadow is not present for the project cards, and the previous reviewer didn't suggested this as well. Kindly fix this so that all project cards look exactly like the Figma design as suggested by the previous reviewer.

Your design | Given design 
---| ---|
![image](https://github.com/Srilearns/Portfolio-Website-Mobile-First/assets/106119105/91a5a4b1-170e-480b-84ed-62ca0e7f9a7f) | ![image](https://github.com/Srilearns/Portfolio-Website-Mobile-First/assets/106119105/e987a640-18d9-495f-a475-7e081eb31104)

_Kindly apply this for all cards_

Reply:
Fixed.